Update S3 Source to be EU compatible (#3946)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md

> var Objects = require('objects-node');

Author: Is there any updates we need to make here?

var analytics = new Analytics(process.env.write_key);

> // NOTE: if you are sending to an EU based source, analytics will need to be initialized

Author: ```suggestion
// NOTE: if you are sending to an EU based source, `analytics-node` will need to be initialized
```

> var analytics = new Analytics(process.env.write_key);
> 
> // NOTE: if you are sending to an EU based source, `analytics-node` will need to be initialized
> // with EU tracking API URL.
> // var analytics = new Analytics(process.env.write_key, {
> //   host: "https://events.eu1.segmentapis.com"
> // });
> 
> var objects = new Objects(process.env.write_key);

Author: ```suggestion
var analytics = new Analytics(process.env.write_key);
var objects = new Objects(process.env.write_key);

// NOTE: if you are sending to an EU-based source, analytics and object SDKs
// need to be initialized with the appropriate host name.
// var analytics = new Analytics(process.env.write_key, {
//   host: "https://events.eu1.segmentapis.com"
// });
// var objects = new Objects(process.env.write_key, {
//   host: "http://objects.euw1.segment.com"
// });

```

var analytics = new Analytics(process.env.write_key);
var objects = new Objects(process.env.write_key);

> // NOTE: if you are sending to an EU-based source, analytics and object SDKs

Author: ```suggestion
// NOTE: if sending to an EU-based source, then the analytics and object SDKs
```

// need to be initialized with the appropriate host name.
// var analytics = new Analytics(process.env.write_key, {
//   host: "https://events.eu1.segmentapis.com"
// });
// var objects = new Objects(process.env.write_key, {

> //   host: "http://objects.euw1.segment.com"

Reviewer: @mericsson Should we ship these two together? Or are the ok to go out individually?

> //   host: "http://objects.euw1.segment.com"

Author: OK to ship individually

var analytics = new Analytics(process.env.write_key);
var objects = new Objects(process.env.write_key);

// NOTE: if you are sending to an EU-based source, analytics and object SDKs
// need to be initialized with the appropriate host name.
// var analytics = new Analytics(process.env.write_key, {
//   host: "https://events.eu1.segmentapis.com"
// });
// var objects = new Objects(process.env.write_key, {
//   host: "http://objects.euw1.segment.com"
// });

> 

Author: ```suggestion
```
